src/v/cluster/partition_balancer_planner.h

```diff
@@ -63,12 +64,15 @@ class partition_balancer_planner {
 
 private:
     struct reallocation_request_state {
-        uint64_t planned_moves_size = 0;
-        absl::flat_hash_map<model::ntp, size_t> ntp_sizes;
+        std::vector<model::node_id> all_nodes;
         absl::flat_hash_map<model::node_id, node_disk_space> node_disk_reports;
         absl::flat_hash_set<model::node_id> unavailable_nodes;
+
+        absl::flat_hash_map<model::ntp, size_t> ntp_sizes;
+
```

Comment:
nit: this change seems unrelated i.e. moving the variables order

Reply:
Yeah I was adding a new field and decided to rearrange fields to my liking a bit :)
Honestly I don't think this merits a separate commit